docs/repositories-configure/configuring-code-patterns.md

```diff
@@ -165,12 +165,6 @@ The table below lists the configuration file names that Codacy detects and suppo
     <td><code>.credo.exs</code>, <code>config/.credo.exs</code></td>
     <td></td>
   </tr>
-  <tr>
```

Comment:
@nicklem We should remove CSSLint here, right? Can you please confirm? 🙏

Reply:
@claudiacarpinteiro @nicklem Please keep all listed tools ([as returned by the api](https://api.codacy.com/api/api-docs#listtools)) and mark the deprecated ones with some visual cue (an asterisk , put it in a separate section below or something else).

Reply:
@codacy-vrhpires I kept the name of the deprecated tools that are still accepted by the Codacy configuration file [on this separate section](https://github.com/codacy/docs/pull/1956/files#diff-6e1b2afc217b0798259f3b0065e205a68fd5c13f20e03dae4014a2c39509173aR140-R151). 

From other tables where we list the **officially supported tools**, we have been removing the tools we deprecate. For example, the [Supported languages and tools](https://docs.codacy.com/getting-started/supported-languages-and-tools/).

Reply:
@claudiacarpinteiro it's great as it is.
Although I have a food for thought: shouldn't we ease the user pointing the replacement tool for each deprecated one?
Let me know what you think and I'll hold the review until then.

Reply:
@codacy-vrhpires Thank you for the input. I added information for deprecated tools replacement. Let me know if you agree.